src/item.cpp

```diff
@@ -8724,11 +8725,17 @@ bool item::use_charges( const itype_id &what, int &qty, std::list<item> &used,
         } else if( e->count_by_charges() ) {
             if( e->typeId() == what ) {
 
+                // Unseal the pocket and parent pockets.
+                if( parent != nullptr ) {
+                    map_cursor mc( pos );
+                    avatar &player = get_avatar();
+                    item_location container( mc, parent );
```

Comment:
I think this assumes that parent is the outermost container on the map. Is that the case here?

Reply:
No guarantee of that. Was having a similar thought this morning. So yeah, this is flawed.

Reply:
Did some more testing and it does not appear that the `item_location` created with the `map_cursor` is a problem. Put some sealed jars of peanut butter in a sling pack, put the sling pack inside a backpack, and put it on a counter. The container was unsealed after making a sandwich with the PB.

I'm not aware of a better way to create an `item_location` for items that may or may not be in inventory, and this appears to be working fine. I see no reason to change it.